wiki/Legal/Copyright/de.md

```diff
@@ -0,0 +1,61 @@
+---
+legal: true
+---
+
+# Urheberrechtspolitik von osu!
+
+osu! ist ein stark an der Community orientiertes Spiel. Alle Spiellevel (Beatmaps) werden durch Community-Mitglieder erstellt und hochgeladen, oft ohne korrekte Einholung der Erlaubnis der jeweiligen Rechteinhaber. Wir tun unser Bestes, Nutzerinhalte nicht zu werben oder daraus Gewinn zu erzielen. Das Spiel wird vollständig aus Nutzerspenden finanziert.
+
+Ein Großteil der Spenden, die wir erhalten, werden für die [Musiklizenzierung](/wiki/Legal/Music_licensing) genutzt. Anfragen zur Lizenzvergabe können an [music@ppy.sh](mailto:music@ppy.sh) gesendet werden.
+
+In Ermangelung dessen hat osu! die folgende Leitlinie bezüglich Urheberrechtsverletzungen in Übereinstimmung mit dem Digital Millennium Copyright Act, 17 U.S.C. 512 verabschiedet.
+
+Die Kontaktinformationen des Beauftragten von osu! für die Entgegennahme von Mitteilungen über geltend gemachte Rechtsverletzungen (im folgenden Beauftragter genannt) wird am Ende dieser Leitlinie aufgelistet und ist beim United States Copyright Office (<https://www.copyright.gov/dmca-directory/>) hinterlegt. osu! nimmt Urheberrecht und andere Rechte an geistigem Eigentum sehr ernst.
+
+Es ist die Politik von osu!,
+
+1. den Zugriff auf oder den Zugang zu Inhalten schnellstmöglich zu sperren oder zu entfernen, von denen es in gutem Glauben annimmt, dass sie Materialien enthalten, die das Urheberrecht Dritter verletzen und
+2. den Dienst für Wiederholungstäter zu entfernen oder einzustellen.
+
+## Urheberrechtsverstöße melden
+
+Wenn du glaubst, dass Inhalte, die sich auf der osu!-Webseite/dem Dienst befinden oder über diese/diesen zugänglich sind, dein Urheberrecht verletzen, dann sende bitte eine Mitteilung über eine angebliche Urheberrechtsverletzung mit den folgenden Informationen an den unten aufgeführten Beauftragten (ziehe einen Rechtsbeistand zu Rate oder verweise auf 17 U.S.C. 512(c), um diese Anforderungen zu bestätigen):
```

Comment:
Somehow both is weird to me 🤔 

The informal form, because obviously it's a legal text. The formal one because it's never used elsewhere besides the legal pages maybe.

I think preferring the formal one here is still better. Could change that.